docs/serving/autoscaling/README.md

```diff
@@ -1,22 +1,17 @@
 # Autoscaling
 
-One of the main features of Knative is automatic scaling of replicas for an application to closely match incoming demand, including scaling applications to zero if no traffic is being received.
-Knative Serving enables this by default, using the Knative Pod Autoscaler (KPA).
-The Autoscaler component watches traffic flow to the application, and scales replicas up or down based on configured metrics.
+Knative Serving provides automatic scaling, or _autoscaling_, for applications to match incoming demand. This is provided by default, by using the Knative Pod Autoscaler (KPA).
```

Comment:
nit: Should we include the link to autoscaler types here?

Reply:
Not yet 🙂 I have some docs to add for that at another time